R/editing.R

```diff
@@ -447,16 +447,25 @@ publish_update <- function(mn,
   # Replace packageId
   doc$packageId <- metadata_updated_pid
 
-  # Replace system if needed
-  if (is.null(doc$system)) {
-    doc$system <- "https://search.dataone.org"
+  # Add landing page
+  if (use_doi == T){
+    doc$dataset$distribution$online$url <- paste0("http://doi.org/", metadata_updated_pid)
```

Comment:
just checking if it matters if it is http or https?

Reply:
not in this case. It matters in the access policy of the system metadata but not here. Good eye!